doc/changes/unreleased.md

```diff
@@ -7,7 +7,9 @@ which is specified in the `noxconfig.py`
 
 ## 📚 Documentation
 * Updated getting_started.rst for allowing tag-based releases
+* [#382](https://github.com/exasol/python-toolbox/issues/382): Refactored `dependency:licenses` to use pydantic models & `poetry show`
```

Comment:
Is this really a documentation issue. Looks more like refactoring

Reply:
Whoops, definitely a refactoring